hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/DeleteJdbcSqlConnectorTest.java

```diff
@@ -0,0 +1,147 @@
+/*
+ * Copyright 2021 Hazelcast Inc.
+ *
+ * Licensed under the Hazelcast Community License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://hazelcast.com/hazelcast-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.sql.impl.connector.jdbc;
+
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import static com.hazelcast.jet.sql.impl.connector.jdbc.JdbcSqlConnector.OPTION_JDBC_URL;
+
+public class DeleteJdbcSqlConnectorTest extends JdbcSqlTestSupport {
```

Comment:
Missing standard test annotations

Reply:
They get inherited from the `SqlTestSupport` and `SimpleTestInClusterSupport`.

```
mvn test -PparallelTest
```
correctly runs them in the following phase:
```
[INFO] --- maven-surefire-plugin:2.22.2:test (default-test) @ hazelcast-sql ---
```
which uses multiple forks.